app/models/page_view.rb

```diff
@@ -71,4 +77,8 @@ def article_searchable_text
   def article_tags
     article.decorate.cached_tag_list_array
   end
+
+  def tweet_tag_refresh_stats
+    TweetTagRefreshStatsJob.perform_later(article.id)
```

Comment:
Hey, sorry about the delay!

I think the final thing we'd want here is a guard clause here that ensures no extra work needs to be done if the article doesn't have a tweet. It doesn't need to be a perfect clause so I think checking for whether the `article.body_markdown` includes `{% tw` (or `{%tw`) or something along those lines would be good enough to ensure we don't run `TweetTagRefreshStatsJob` job too often.